pygac/reader.py

```diff
@@ -590,15 +647,15 @@ def get_lonlat(self):
             self.lons, self.lats = self._get_lonlat()
             self.update_meta_data()
 
+            # Adjust clock drift
```

Comment:
What is the effect of adjusting clock drift before interpolation?

Reply:
it means we don't need to interpolate all 2048 (or 409) values for each scanline, but rather just interpolate the 51 subsampled original values. In the end, I don't think it matter for precision, rather it's a performance boost.